### ISSUE TITLE
Thank You Email Template to sponsors

### ISSUE THREAD

**Issue by Aycrazy** [codeformilwaukee/hack-night-planning#82]
Don't have to worry about Amanda this time around (I already sent her something), but we need to have a template for these thank yous can @Dannycap and/or @anthonyjesmok  draft his up for review?

**Comment by anthonyjesmok** [comment]
Do we still need this?

**Comment by Aycrazy** [comment]
Yes. Never had a template. Definitely an initial task of new financial lead

**Comment by BarkleyBG** [comment]
Im thinking we should write thank you cards for presenters and sponsors.

We could bring a set of cards to the event where all Core Team signs them, for example
